pkg/minikube/service/service.go

```diff
@@ -265,22 +264,24 @@ func PrintServiceList(writer io.Writer, data [][]string) {
 	table.Render()
 }
 
-// WaitAndMaybeOpenService waits for a service, and opens it when running
-func WaitAndMaybeOpenService(api libmachine.API, namespace string, service string, urlTemplate *template.Template, urlMode bool, https bool,
-	wait int, interval int) error {
+// WaitForService waits for a service, and opens it when running
```

Comment:
Make comment accurate again

Reply:
This method no longer opens a URL.

Reply:
Comment has been updated to reflect returning url instead of opening in browser. PR title has been modified too.